src/feautures/MovieList/MovieTile/Rates/styled.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+import { ReactComponent as Star } from "./star.svg";
+
+export const StyledRates = styled.div`
+    margin: 8px 0;
```

Comment:
I would change margin to: `margin-top:  39px;` referring to figma

Reply:
There is no such think in figma... 
But maybe you got a point and better to stick rates to bottom border

Reply:
![obraz](https://user-images.githubusercontent.com/118211320/220937531-c549e8bf-8355-4501-912c-9e4facf9331f.png)

That is the margin I talk about 😉

Reply:
Check again in diferent tile where there is no tags, or subtitle.. 
So its be better to stick it to bottom, and left 4px margin as it is for media..

Reply:
You're right. It can be different above so sticking to the bottom is better option